Resources/Private/Templates/Backend/Library/Upgrade.html

```diff
@@ -0,0 +1,32 @@
+{namespace neos=Neos\Neos\ViewHelpers}
+<f:layout name="BackendSubModule" />
+
+<f:section name="content">
+    <div class="neos-row-fluid">
+
+        <script>
+            window.H5PAdminIntegration = {adminSettings -> f:format.raw()};
+            window.H5PIntegration = {coreSettings -> f:format.raw()};
+        </script>
+
+        <f:for each="{coreScripts}" as="script">
+            <script src="{script}" charset="UTF-8"></script>
+        </f:for>
+        <script src="{h5pVersionScriptUrl}" charset="UTF-8"></script>
+        <script src="{h5pContentUpgradeScriptUrl}" charset="UTF-8"></script>
+        <script src="{h5pUtilsScriptUrl}" charset="UTF-8"></script>
+
+        <legend>{library.title}</legend>
+
+        <div id="h5p-admin-container">Please enable JavaScript.</div>
+
+        <div class="neos-footer">
+            <neos:link.module
+                title="Back to overview"
+                path="h5p/library"
+                action="index"
+                class="neos-button">
+                Back
```

Comment:
I decided to place a back button in the template instead of letting the upgrade-script running in the client render a back-link or back-button when the upgrading is done.

Only drawback imo (plz review my thought process): When an upgrade is in progress (percentage counting up) and the user presses back at the same time, the upgrade takes place but no user feedback is given (no "x contents were successfully upgraded"-screen is rendered).